src/components/atoms/ScreenTitle/ScreenTitle.tsx

```diff
@@ -0,0 +1,9 @@
+import {Text} from 'react-native';
+import {styles} from './styles';
+
+type TScreenTitleProps = {
+  children: string;
+};
+export const ScreenTitle = ({children}: TScreenTitleProps) => {
+  return <Text style={styles.title}>{children}</Text>;
```

Comment:
polecałbym custom Text component, wtedy typografię sobie ogarniesz, czyli warianty font colorów i rozmiarów, bez tego będziesz z palce te fontSizes ustalał bez standardów żadnych i gówno będzie

Reply:
to sobie zrobię jako kolejne niech na razie tak pójdzie